iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
+      return
+    }
+
+    let titleAttributes: [NSAttributedString.Key: Any] = [
+      .font: titleFont,
+      .foregroundColor: UIColor.blackPrimaryText()
+    ]
+
+    let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+
+    guard let secondaryTitle = secondaryTitle,
+          let secondaryTitleFont = StyleManager.shared.theme?.fonts.medium16 else {
+      titleLabel?.attributedText = attributedText
+      return
+    }
+
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.paragraphSpacingBefore = 8
+    let secondaryTitleAttributes: [NSAttributedString.Key: Any] = [
+      .font: secondaryTitleFont,
+      .foregroundColor: UIColor.blackPrimaryText(),
+      .paragraphStyle: paragraphStyle
+    ]
+
+    attributedText.append(NSAttributedString(string: "\n" + secondaryTitle, attributes: secondaryTitleAttributes))
+    titleLabel?.attributedText = attributedText
```

Comment:
Can the logic be simplified to avoid copy-paste and unnecessary returns?

Reply:
It's possible to put all secondary title logic inside if let context, but it will increase [pyramid of doom](https://levelup.gitconnected.com/escape-the-pyramid-of-doom-c58edd326225). I prefer early-return programming style. Each previous time `titleLabel?.attributedText` is assigned - return is invoked, so we don't have any overhead in runtime.